src/main/controllers/ListController.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from 'express';
```

Comment:
should we rename from `listController`? its quite vague as a list

Reply:
Updated to hearingListController :)